velox/type/TimestampConversion.h

```diff
@@ -134,4 +134,12 @@ inline Timestamp fromTimestampString(const StringView& str) {
 
 Timestamp fromDatetime(int64_t daysSinceEpoch, int64_t microsSinceMidnight);
 
+inline Timestamp fromUTCMillis(int64_t millis) {
```

Comment:
Can we use `Timestamp::fromMillis` here?

Reply:
Was about to update. As my implemenation didn't cover -ve millis/micros